Fix typos in `src/lib/OpenEXR/`

luzpaz opened the pull request:
Fix typos in source comments and doxygen content. Found via `codespell -q 3 -S CHANGES.md,./ASWF/tsc-meetings -L dout,nnumber,offsetp`

meshula commented on src/lib/OpenEXR/ImfFastHuf.cpp:
```diff
@@ -507,7 +507,7 @@ FastHufDecoder::refill (
     // We can have cases where the previous shift of bufferBack is << 64 -
     // this is an undefined operation but tends to create just zeroes.
-    // so if we won't have any bits left, zero out bufferBack insetad of computing the shift
+    // so if we won't have any bits left, zero out bufferBack insread of computing the shift
```
```suggestion
    // so if we won't have any bits left, zero out bufferBack instead of computing the shift
```

meshula reviewed (approved):
Thanks for the spell check!

xlietz commented:
@luzpaz this commit needs to be properly signed off in order to merge. Directions on how to fix this are under "Some checks were not successful" - scroll down to the DCO line then click on the "Details" link to the right. Thanks!

luzpaz committed:
Fix typos

Fix typos in source comments and doxygen content. Found via `codespell -q 3 -S CHANGES.md,./ASWF/tsc-meetings -L dout,nnumber,offsetp`
Update src/lib/OpenEXR/ImfFastHuf.cpp

Signed-off-by: luz paz <luzpaz@github.com>
Signed-off-by: Nick Porcino <meshula@hotmail.com>

luzpaz commented:
Squashed and signed commit.

cary-ilm commented:
Thanks, good catches. A little embarrassing, actually.
